xxh3.h

```diff
@@ -540,7 +540,7 @@ static void XXH3_scrambleAcc(void* acc, const void* key)
         xacc[i] ^= xacc[i] >> 47;
 
         {   U64 const p1 = XXH_mult32to64(xacc[i] & 0xFFFFFFFF, xkey[left]);
-            U64 const p2 = XXH_mult32to64(xacc[i] >> 32,        xkey[right]);
+            U64 const p2 = XXH_mult32to64(xacc[i] >> 32,        xkey[right]); // warning C5045, Spectre related - may decrease the speed with some circumstances (the /Qspectre option may be turned on by default and a fence instruction may appear = UD on older processors)
```

Comment:
Not sure if we can do much about that.

Reply:
there may be some defines to avoid the Spectre fix, however I did not dig into it yet

Reply:
I wonder what kind of Spectre issue could be raised at this line.
I don't see any branch nor speculation that could be used to trigger a side effect ...

Reply:
I will check the particular asm code to investigate. There's a loop with the arithmetics (so at least one branch is there) and you never know what the compiler decides to do to speed it up.